ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -213,21 +244,152 @@ def deploy_workload(self):
         # TODO
         # drpc_yaml_file needs to be committed back to the repo
         # because ACM would refetch from repo directly
-
         # load channel.yaml
         channel_yaml_data = templating.load_yaml(self.channel_yaml_file)
         channel_yaml_data["spec"]["pathname"] = self.workload_repo_url
         templating.dump_data_to_temp_yaml(channel_yaml_data, self.channel_yaml_file)
-
         # Create the resources on Hub cluster
         config.switch_acm_ctx()
         run_cmd(f"oc create -k {self.workload_subscription_dir}")
         run_cmd(f"oc create -k {self.workload_subscription_dir}/{self.workload_name}")
+
         if self.is_placement:
             self.add_annotation_to_placement()
             run_cmd(f"oc create -f {self.drcp_data_yaml.name}")
         self.verify_workload_deployment()
 
+    def deploy_workloads_on_managed_clusters(
+        self, primary_cluster=True, secondary_cluster=False
+    ):
+        """
+        Deployment specific to busybox workload on both primary and secondary clusters
+
+        Args:
+            primary_cluster(bool) : True if apps needs to be deployed on primary cluster
+            secondary_cluster(bool) : True if apps needs to be deployed on secondary cluster
+
+        """
+        self._deploy_prereqs()
+        workload_namespaces = []
+
+        # By default, it deploys apps on primary cluster if not set to false
+        clusters = [self.preferred_primary_cluster] if primary_cluster else []
+        if secondary_cluster:
```

Comment:
If I want to deploy workloads on `secondary_cluster` which is set to False by default, the function would fail because clusters will be equal to [] empty list and we can not iterate over it. We should handle this use case.

Reply:
If you will set the parameters like:
```
primary_cluster=False
secondary_cluster=True
```
than it will set the `clusters` variable to empty list on the first line (276) and then add the secondary cluster to the empty list in `clusters`, which then will be used in the following `for` loop.

Reply:
But where are we passing the secondary cluster to the empty list?

Reply:
on the following line:
```
clusters.append(self.preferred_secondary_cluster)
```